[2023-10-22T12:29:00.497+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL.Upload_Data manual__2023-10-22T12:28:59.131904+00:00 [queued]>
[2023-10-22T12:29:00.506+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL.Upload_Data manual__2023-10-22T12:28:59.131904+00:00 [queued]>
[2023-10-22T12:29:00.506+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-22T12:29:00.517+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): Upload_Data> on 2023-10-22 12:28:59.131904+00:00
[2023-10-22T12:29:00.522+0000] {standard_task_runner.py:57} INFO - Started process 1673 to run task
[2023-10-22T12:29:00.525+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ETL', 'Upload_Data', 'manual__2023-10-22T12:28:59.131904+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/first_dags.py', '--cfg-path', '/tmp/tmpmesjqoi0']
[2023-10-22T12:29:00.527+0000] {standard_task_runner.py:85} INFO - Job 21: Subtask Upload_Data
[2023-10-22T12:29:00.564+0000] {task_command.py:416} INFO - Running <TaskInstance: ETL.Upload_Data manual__2023-10-22T12:28:59.131904+00:00 [running]> on host dcd154787c99
[2023-10-22T12:29:00.633+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='me' AIRFLOW_CTX_DAG_ID='ETL' AIRFLOW_CTX_TASK_ID='Upload_Data' AIRFLOW_CTX_EXECUTION_DATE='2023-10-22T12:28:59.131904+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-22T12:28:59.131904+00:00'
[2023-10-22T12:29:00.634+0000] {sql.py:274} INFO - Executing: 
        SELECT current_database();

        
[2023-10-22T12:29:00.640+0000] {base.py:73} INFO - Using connection ID 'postgres_docker' for task execution.
[2023-10-22T12:29:00.649+0000] {base.py:73} INFO - Using connection ID 'postgres_docker' for task execution.
[2023-10-22T12:29:00.659+0000] {sql.py:418} INFO - Running statement: 
        SELECT current_database();

        , parameters: None
[2023-10-22T12:29:00.661+0000] {sql.py:427} INFO - Rows affected: 1
[2023-10-22T12:29:00.686+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ETL, task_id=Upload_Data, execution_date=20231022T122859, start_date=20231022T122900, end_date=20231022T122900
[2023-10-22T12:29:00.738+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-22T12:29:00.806+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
